src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -288,6 +288,21 @@ public function where($column, $operator = null, $value = null, $boolean = 'and'
         return $this;
     }
 
+    /**
+     * Add a basic where Like clause to the query, and return the All Like results.
+     *
+     * @param  \Closure|string|array|\Illuminate\Database\Query\Expression  $column
+     * @param  mixed  $value
+     * @return \Illuminate\Database\Eloquent\Model|static|null
+     */
+    
+    public function whereLike($column, $value = null)
+    {
+        $this->where($column,'LIKE','%'.$value.'%');
```

Comment:
```suggestion
        $this->where($column, 'LIKE', $value);
```

I would remove the `%` because sometimes you want to search only for `xxx%` or `%xxx`. I would expect that the `LIKE` Operator is set only and not that the value is changed.

Reply:
so let's figure a way to let us decide from the method argument